index.html

```diff
@@ -0,0 +1,305 @@
+                    <li>html</li>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button class="cardbut">see project</button>
+                <img src="img/hand-cursor 1.png" alt="cursor">
+            </div>
+            
+        </div>
+
+        <div class="section4">
+            <div class="tonicpart">
+                <h2>
+                    Tonic
+                </h2>
+                <ul class="firstlist">
+                    <li>Uber</li>
+                    <li>lead Developer</li>
+                    <li>2018</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups requied</p>
+                <ul class="secondlist1">
+                    <li>html</li>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button class="cardbut">see project</button>
+                
+            </div>
+            <div class="portimage">
+                <img src="img/Snapshoot Portfolio4.png" alt="nature">
+            </div>
+        </div>
+     </section>
+<!-- About myself -->
+     <div class="aboutme">
+        <div class="firstside">
+            <h2>
+                About <br> Myself
+            </h2>
+            <p>
+                Hello i'm a telecommunications and Software Engineer. I can help you build a product feature or website. Look through some of my work and experience if you like what you see and have a project you need coded, don't hesitate to contact me
+            </p>
+            <div class="connect">
+                <div class="text3">
+                    <div class="text3">
+                        <a href="#"> LET'S CONNECT 
+                        </a>
+                  </div>
+                </div>
+                <div>
+                    <ul class="links">
+                        <li>
+                            <a href="https://twitter.com/Taku16671414">
+                                <img src="img/twitter.png" alt="twitter">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://www.linkedin.com/in/shakespear-takudzwa-samu-b61b38a3">
+                                <img src="img/Linkedin icon.png" alt="linkedin">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="img/messager.png" alt=" messages">
+                            </a>
+                        <li>
+                            <a href="#">
+                                <img src="img/cat.png" alt="CAT">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
```

Comment:
- [ ] Kindly put more effort to ensure that your design is very close to the figma template, this makes hiring managers to have more confidence in us that we can reproduce a replica of whatsoever design template given. 👍 

## Your design

![A2](https://user-images.githubusercontent.com/103386183/199671557-4869b10e-bc4e-4b57-b739-44292e52b121.png)


## Figma design

![A4](https://user-images.githubusercontent.com/103386183/199671627-5c3dcb54-a13e-45f6-93cb-34cc9fde910d.png)

Reply:
## issue resolved 
thank you for that kind suggestion, I will make sure I pay more attention to details next time
- I managed to fix te issue by making sure that the two sections share the same space 
- and also using margin to properly align everything